src/Psy/Command/TimeitCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Psy\Command;
+
+use Psy\Configuration;
+use Psy\Shell;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+
+/**
+ * Class TimeitCommand
+ * @package Psy\Command
+ */
+class TimeitCommand extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('timeit')
+            ->setDefinition(array(
+                new InputArgument('target', InputArgument::REQUIRED, 'A target object or primitive to profile.', null),
```

Comment:
I've been thinking about this a bit, and the more I consider it the more I feel that we need a different input string parsing type.

Right now, all command input is parsed like command line args. We would really be better served by a hybrid between command line args and "just give me the string they typed". Here, for example, if you're trying to time `foo($bar, 'baz')` you'll run into problems, because InputArgument will just grab part of that as `target` :-/